### DISCUSSION TITLE
Floppy not found?

### DISCUSSION THREAD

**Discussion by kayodde** [86Box/86Box#4807] [General]
Trying out 86box for the first time, I can't seem to figure out why but none of the floppy files and .iso files I try and insert into the floppy and CD drives are being detected. What am I doing wrong? It looks for them and then just says not found and to insert a boot diskette into A:
I am on version 4.2.1. I downloaded the build from the 86box website and everything else seems to work fine including hard drive detection.

**Comment by eddmanx** [comment]
Set the correct FDD type in the BIOS.
Only OEM 98 discs are self-bootable. For other 95/98 releases use a floppy boot disc.

**Comment by hkevinli** [comment]
Please post your 86box.cfg file.
